src/execution_tree/primitives/base_primitive.cpp

```diff
@@ -4124,6 +4282,24 @@ namespace phylanx { namespace execution_tree
             }
             return os;
 
+        case 8:     // phylanx::ir::dictionary
+        {
+            os << "dict{";
+            bool first = true;
+            for (auto const& elem : util::get<8>(val))
+            {
+                if (!first)
+                {
+                    os << ", ";
+                }
+                first = false;
+                ast::detail::to_string{os}(elem.first);
+                os << ": ";
+                ast::detail::to_string{os}(elem.second);
+            }
+            os << "}";
+        }
+            return os;
```

Comment:
Could align the formatting of this `case` with the rest of the function, please?